lib/game.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'player'
+
+class Game
+
+  def initialize(player_one, player_two)
+    @player_one_name = player_one.name
+    @player_two_name = player_two.name
+
+    @player_one_selection = player_one.selection
+    @player_two_selection = player_two.selection
+  end
+
+  def result
+    if @player_one_selection == @player_two_selection
+      "It's a Draw!"
+    elsif (@player_one_selection == "Rock" && @player_two_selection == "Scissors") ||
```

Comment:
This part of the code could be made a bit easier to read by using early returns (like here: https://stackoverflow.com/a/37880774) - you could have something like:
```ruby
return "#{@player_one_name} wins" if @player_one_selection == "Rock" && @player_two_selection == "Scissors"
return "#{@player_one_name} wins" if @player_one_selection == "Paper" && @player_two_selection == "Rock"
return "#{@player_one_name} wins" if @player_one_selection == "Scissors" && @player_two_selection == "Paper"
```

Since we're using OR (||), we'll return that player one wins as soon as one of those conditions is true, so the result is the same